resources/js/components/About.vue

```diff
@@ -66,6 +66,20 @@
                         </div>
                     </template>
                 </AboutCollapsible>
+                <AboutCollapsible class="scroll-mt-12 border-b-1 border-gray-softest">
+                    <template v-slot:summary>{{ $t('Settings') }}</template>
+                    <template v-slot:content>
+                        <div class="align-center flex">
```

Comment:
Now I see there there's a misnamed class

```suggestion
                        <div class="items-center flex">
```